docs/book/src/developer/release-support-guidelines.md

```diff
@@ -34,8 +34,10 @@ standard [backport policy](https://github.com/kubernetes-sigs/cluster-api/blob/m
 
 | Minor Release | API Version | Supported Until                                    |
 |---------------|-------------|----------------------------------------------------|
-| v0.7.x        | **v1beta2** | when v0.9.0 will be released                       |
-| v0.6.x        | **v1beta2** | when v0.8.0 will be released                       |
+| v0.9.x        | **v1beta2** | when v1.1.0 will be released                       |
+| v0.8.x        | **v1beta2** | when v1.0.0 will be released                       |
```

Comment:
Is `v1.0.0` for the next CAPIBM release?
We have marked the next version as `v0.10.0` right?

Reply:
missed it, fixed.